venter/admin.py

```diff
@@ -96,41 +97,32 @@ def send_emails(modeladmin, request, queryset):  # pylint: disable=R0201
         mailing_list = []
 
         for item in queryset:
-            # Check if the complaint has a valid authority set
-            if item.authority_email is None:
-                continue
-
             input_list = [i for i in ComplaintMedia.objects.filter(complaint=item.id).values('image_url')]
             output_list = [images[key] for images in input_list for key in images]
 
             subject = f'Complaint from {item.created_by} on {item.report_date:%A, %d %b %Y at %I:%M %p}'
 
             # Checks for attachments and modifies the email message based on that
-            if output_list:
-                message = (
-                    f'Complaint Description: {item.description}\n'
-                    f'Location Description: {item.location_description}\n'
-                    f'Status: {item.status}\n'
-                    f'Images: {output_list}'
-                )
-
-            elif not output_list:
-                message = (
-                    f'Complaint Description: {item.description}\n'
-                    f'Location Description: {item.location_description}\n'
-                    f'Status: {item.status}\n'
-                )
+
+            message = (
+                f'Complaint Description: {item.description}\n'
+                f'Location Description: {item.location_description}\n'
+                f'Status: {item.status}\n'
+                f'Images: {", ".join(output_list)}'
```

Comment:
Wrap this in a ternary or something so "Images" will disappear when there are no images (regression on this)

Reply:
I had an if-else statement that dealt with that. Is it okay if we use two separate formats?

Reply:
Never mind, I just used a simple if condition to join them together

Reply:
Great!